dashboards/datasources.yml

```diff
@@ -9,5 +9,5 @@ datasources:
     orgId: 1
     type: prometheus
     access: proxy
-    url: http://host.docker.internal:9090
+    url: http://prometheus:9090
```

Comment:
I don't think this works with host mode, does it? Have you tried it on Linux?

Reply:
You're right. I've update the linux compose file to make this work. Tested on linux.